### DISCUSSION TITLE
NAS

### DISCUSSION THREAD

**Discussion by grssll** [blakeblackshear/frigate#13995] [Ask A Question]
When adding NAS I'm unable to rename frigate folder to frigate_tmp.
The action cant be completed as the folder is open in another program.
Addon is not running, and I followed the instructions from WIKI.
Thanks

**Comment by NickM-27** [comment]
maybe try restarting the whole host and doing it after (make sure frigate is not set to autostart)

**Comment by grssll** [comment]
I have tried all the options.
Deleted frigate addon and frigate integration.
Restarted HA but for some reason still have frigate folder in media.
Unable to delete or rename it.
